phpBB/styles/prosilver/theme/cp.css

```diff
@@ -300,13 +300,6 @@ ol.def-rules li {
 	padding: 0 3px;
 }
 
-/* DEPRECATED 3.2.6
-.pmlist li.pm_message_reported_colour, .pm_message_reported_colour {
-	border-left-color: transparent;
-	border-right-color: transparent;
-}
-*/
-
```

Comment:
@hanakin this is the one im referring about in my last message

Reply:
this can be removed IMHO, it is akready commented code so unusable, the same class exists elsewhere but not used, still
`"ROOT/3211\styles\prosilver\theme\colours.css"(905,2)`

Reply:
yeah i noticed a few that are blank i'll make a PR for the changes i just reverted and the ones that are blank

Reply:
Yes it's fine here cause it was commented out

Reply:
ok cool ill make a new ticket to remove the uncommented not used 3.2 classes for master